### PR TITLE
Remove check boxes on channels page

### PR DESCRIPTION
**Pull request by hodgestar**
- On the channels page: https://go.vumi.org/channels/
- selecting the checkboxes and then clicking the 'release' button in the header doesn't do anything.
- this is freaking out our beta users
- let's kill the checkboxes and release button and remove them from the UI

screen shot 2013-08-29 at 4 55 22 pm

### CODE REVIEW

**Comment by hodgestar** [comment]
I'll change this to match the multi-button thing we use for campaigns.

**Comment by hodgestar** [comment]
I want to factor out the common code among all the action classes but I'm leaving that for #688.

File: go/channel/templates/channel/show.html [hunk 1]
```diff
@@ -28,3 +25,13 @@
...
 </div>
 {% endblock %}
+
+{% block ondomready %}
+    var actions = new go.channel.show.ChannelActionsView({
+        el: '.content .actions .right'
+    });
+
+    $('.content .sub-actions .disabled').popover({placement: 'left'});
+    $('.content .sub-actions .disabled').on('click', function() { return false; });
```
Comment (justinvdm) [inline go/channel/templates/channel/show.html]: I'm not too sure if these lines are needed. I also see them in the router template, which was probably what was being used as a reference, but I'm wondering if both can go?

Comment (justinvdm) [inline go/channel/templates/channel/show.html]: Ah, I see these lines are in the conversation template too. That template has a `.sub-actions` element, so maybe its only relevant to that template?

Comment (hodgestar) [inline go/channel/templates/channel/show.html]: They're not needed but I was intentionally keeping them for consistency (and to save someone the trouble of adding them if they added an action that needed them).

Comment (justinvdm) [inline go/channel/templates/channel/show.html]: I see. I think we should be avoiding returning `false` in ui event callbacks (it can have unintended side-effects), and should use `e.preventDefault` instead. Since this is scattered all over the place, maybe we should leave that for a different PR.

**Comment by justinvdm** [comment]
:+1:

**Comment by justinvdm** [comment]
Another :+1: :)
